examples/chat/client/cpp/chat.cc

```diff
@@ -14,29 +14,31 @@
  * limitations under the License.
  */
 
+#include <openssl/evp.h>
+
+#include <iomanip>
+#include <sstream>
 #include <thread>
 
 #include "absl/base/thread_annotations.h"
 #include "absl/flags/flag.h"
 #include "absl/flags/parse.h"
+#include "absl/strings/escaping.h"
 #include "absl/synchronization/mutex.h"
 #include "examples/chat/proto/chat.grpc.pb.h"
 #include "examples/chat/proto/chat.pb.h"
 #include "glog/logging.h"
 #include "include/grpcpp/grpcpp.h"
 #include "oak/client/application_client.h"
-#include "oak/common/nonce_generator.h"
+#include "oak_abi/proto/identity.pb.h"
 
 ABSL_FLAG(bool, test, false, "Run a non-interactive version of chat application for testing");
 ABSL_FLAG(std::string, address, "localhost:8080", "Address of the Oak application to connect to");
-// TODO(#1357): Use a public / secret key pair instead of bearer token credentials.
-ABSL_FLAG(std::string, room_access_token, "", "Base64-encoded public key of the room to join");
+ABSL_FLAG(std::string, room_secret, "",
```

Comment:
Maybe rename to `room_secret_file` or `room_secret_path` for clarity? Though even `ca_cert` below should be renamed, maybe together at some point in the future.